tasks/logger-file.js

```diff
@@ -0,0 +1,19 @@
+const gulp = require('gulp');
+const pathBuilder = require('../pathBuilder');
+const config = require('../config');
+const fs = require('fs');
+
+/**
+ *  `logger-file-create` Task
+ *  ---------------
+ *
+ */
+
+gulp.task('logger-file-create', () => {
```

Comment:
We've used a slightly different naming convention for a lot of the other tasks.  Might make more sense if this was called `logger:createFile` or something like that.

Reply:
Yup will update.